tests/unit/core/test_installAddon.js

```diff
@@ -601,6 +601,29 @@ describe(__filename, () => {
         .returns(false);
     });
 
+    describe('getAddonEnabledStatus', () => {
+      it('calls addonManager.getAddon()', () => {
+        const fakeAddonManager = getFakeAddonManagerWrapper();
+        let guid;
+
+        const addon = createInternalAddon({
+          ...fakeAddon,
+          _addonManager: fakeAddonManager,
+        });
+        const { root } = renderWithInstallHelpers({
+          ...addon,
+          addon: { guid },
+          _addonManager: fakeAddonManager,
+        });
+
+        const { getAddonEnabledStatus } = root.instance().props;
+
+        return getAddonEnabledStatus().then((addonObject) => {
+          sinon.assert.calledWith(fakeAddonManager.getAddon, addonObject.guid);
+        });
+      });
+    });
```

Comment:
We need a test case for when there is an error (`it('returns false when there is an error'`)